nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/UnpackContent.java

```diff
@@ -200,7 +217,13 @@ public void onScheduled(ProcessContext context) throws ProcessException {
         if (fileFilter == null) {
             fileFilter = Pattern.compile(context.getProperty(FILE_FILTER).getValue());
             tarUnpacker = new TarUnpacker(fileFilter);
-            zipUnpacker = new ZipUnpacker(fileFilter);
+
+            char[] password = null;
+            final PropertyValue passwordProperty = context.getProperty(PASSWORD);
+            if (passwordProperty.isSet()) {
+                password = passwordProperty.evaluateAttributeExpressions().getValue().toCharArray();
```

Comment:
If the PD no longer supports EL, this chained method is also unnecessary.

Reply:
I will also remove this chained property evaluation.